### DISCUSSION TITLE
Read external data from queue

### DISCUSSION THREAD

**Discussion by SquishyPandaDev** [uNetworking/uWebSockets#1554] [Q&A] [answered]
I'm working on project that is a plugin for an app with live data. The websocket server is run in its own thread. The plugin puts incoming data into a standard stl queue. Can I setup uWebSockets to poll this queue and send updated data to clients

**Comment by uNetworkingAB** [comment, marked as answer]
Yes use Loop::defer

**Reply by SquishyPandaDev** [reply]
Thanks. Sorry I missed that

**Reply by uNetworkingAB** [reply]
No problem ☺️
